avocado/core/nrunner/runnable.py

```diff
@@ -487,6 +499,24 @@ def runner_command(self, runners_registry=None):
         """
         return Runnable.pick_runner_command(self.kind, runners_registry)
 
+    @staticmethod
+    def pick_runner_class_from_entry_point_kind(kind):
```

Comment:
My only suggestion here would be to refactor the following method `pick_runner-class_from_entry_point`, to avoid the duplicated code and reuse this code (if possible):

```python
def pick_runner_class_from_entry_point(self):
   return Runnable.pick_runner_class_from_entry_point_kind(self.kind)
```

Reply:
Oh, you're absolutely right.  In fact, I meant to do it, and forgot it.